test-resources/swift/cleanup_rules/expected/SampleClass.swift

```diff
@@ -163,4 +163,56 @@ class SampleClass {
         var value = 2
         var value2 = 3
     }
+
+    func checkIfShortCircuitStatementsWithBooleanPrefix() {
```

Comment:
Let's not add this to this file. Let's make a separate swift test case? This will add up to the slow down. 
Adding a separate test case will enable us to leverage `cargo test`  level parallelism